test.sh

```diff
@@ -1,40 +1,74 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+# -e Stops execution in the instance of a command or pipeline error
+# -u Treat unset variables as an error and exit immediately
+set -eu
+
 # Run all tests and coverage checks. Called from Travis automatically, also
 # suitable to run manually. See list of prerequisite packages in .travis.yml
 if type realpath >/dev/null 2>&1 ; then
-  cd $(realpath $(dirname $0))
+  cd "$(realpath -- $(dirname -- "$0"))"
 fi
 
-set -exo pipefail
-TRAVIS=${TRAVIS:-false}
+#
+# Defaults
+#
+STATUS="FAILURE"
+# RUN, UNIT_FILTER, and INT_FILTER vars are here for backwards compatibility,
```

Comment:
I don't think we need backwards compatibility here; am I missing something?

Reply:
Yeah, I can yank these if we don't want to support any kind of backwards compatibility.